lib/2019.ts

```diff
@@ -1,5 +1,5 @@
 import type {AnySchemaObject} from "./types"
-import AjvCore, {Options} from "./core"
+import AjvCore, {AjvOptions} from "./core"
```

Comment:
Not sure what is the best way to deal with diverging options... It is a bit weird that the type that is used in constructor is different from the type that is exported (in the bottom)... Maybe it is better just to allow this option in all classes and ignore it, at least until the next major version?

As long as it is documented it is JTD only option it is ok I think.

Reply:
Alright, I restored the old Options and documented `timestamp` in a few places.